src/Components/Announcement.vue

```diff
@@ -36,7 +36,6 @@
 					{{ author }}
 					·
 					<span v-if="isScheduled"
-						class="live-relative-timestamp"
```

Comment:
You are just missing the `:data-timestamp="timestamp"` field with the scheduled timestamp?

Reply:
yes, but adding this "resets" the announcement to the timestamp of the post, not the scheduled one